spec/requests/api/v1/private_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::PrivateController, type: :controller do
+  describe 'GET#index' do
+    context 'when user is not sign in' do
+      it 'returns unauthorized error' do
```

Comment:
```suggestion
    context 'when the user is unauthenticated' do
      it 'responds unauthorized status' do
```

Or `given the user does NOT logged in`, or you can propose your own.

Reply:
I will change to `returns unauthorized status`, keeping `returns` to make it consistent with other specs 🙏